content/en/docs/releases/supported-releases/index.md

```diff
@@ -62,6 +62,7 @@ As of now, data plane to data plane is compatible across all versions; however,
 | Version         | Currently Supported  | Release Date      | End of Life            | Supported Kubernetes Versions | Tested, but not supported          |
 | --------------- | -------------------- | ----------------- | ---------------------- | ----------------------------- | ---------------------------------- |
 | master          | No, development only |                   |                        |                               |                                    |
+| 1.17            | Yes                  | February 14, 2023 | ~Sept 2023 (Expected)  | 1.23, 1.24, 1.25, 1.26        | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21 |
```

Comment:
Need to add 1.22 under `Tested` (at least I assume it will be in that column.

Reply:
Yes, will do.